lib/tasks/import.rake

```diff
@@ -0,0 +1,57 @@
+namespace :reports do
```

Comment:
Please add rake comment, something like "This task is only for development purposes, database tables must be clean before importing."